srcs/Config.cpp

```diff
@@ -0,0 +1,91 @@
+#include "Config.hpp"
+
+/* ************************************************************************** */
+/* ---------------------------- STATIC VARIABLE ----------------------------- */
+/* ************************************************************************** */
+
+/* ************************************************************************** */
+/* ------------------------------ CONSTRUCTOR ------------------------------- */
+/* ************************************************************************** */
+
+Config::Config() {}
+Config::Config(std::string config_block, char** env)
+{
+	std::vector<std::string> strs = ft::split(config_block, '\n');
+	this->m_software_name = strs[0].substr(strs[0].find(" ") + 1);
+	this->m_software_version = strs[1].substr(strs[1].find(" ") + 1);
+	this->m_http_version = strs[2].substr(strs[2].find(" ") + 1);
+	this->m_cgi_version = strs[3].substr(strs[3].find(" ") + 1);
+	this->m_base_env = env;
+}
+
+Config::Config(const Config& copy)
+: m_software_name(copy.get_m_software_name()), m_software_version(copy.get_m_software_version()), \
+m_http_version(copy.get_m_http_version()), m_cgi_version(copy.get_m_cgi_version()), m_base_env(copy.get_m_base_env()) {}
+
+/* ************************************************************************** */
+/* ------------------------------- DESTRUCTOR ------------------------------- */
+/* ************************************************************************** */
+
+Config::~Config()
+{
+	this->m_software_name.clear();
+	this->m_software_version.clear();
+	this->m_http_version.clear();
+	this->m_cgi_version.clear();
+	this->m_base_env = nullptr;
```

Comment:
srcs/Config.cpp 36줄: nullptr은 C++11 키워드인 것으로 알고 있습니다.
```suggestion
	this->m_base_env = NULL;
```

Reply:
+11 키워드였군요! 수정하도록 하겠습니다.